lib/settings.js

```diff
@@ -0,0 +1,36 @@
+/* @flow */
+'use strict'
+
+/* ::
+import type {ServerCLISettings} from '../types.js'
+*/
+
+// Use boom for error handling
+const Boom = require('boom')
+
+const ensureEnvVariable = require('./ensure-env-var.js')
+
+function serverCLI () /* : ServerCLISettings */ {
+  const bucket = ensureEnvVariable('SERVER_CLI_BUCKET')
+  const deploymentUrl = ensureEnvVariable('SERVER_CLI_DEPLOYMENT_URL')
+  const region = ensureEnvVariable('SERVER_CLI_REGION')
+  return {
+    bucket,
+    deploymentUrl,
+    region
+  }
```

Comment:
Consider abandoning the temporary variables and just returning the object. You could even make it a short-arrow function e.g.

```js
const serverCLI = () /* : ServerCLISettings */ => ({
  bucket: ensureEnvVariable('SERVER_CLI_BUCKET'),
  // TODO: etc
})
```